File: dotnet-music-app/Models/SongList.cs
public class SongList
{
    public int Id { get; set; }
    public required string Name { get; set; }
    public string ImagePath { get; set; }
    public required List<long> SongIds { get; set; }
    public SongList() { }
    public SongList(int id, string name, string imagePath, List<long> songIds)
    {
        Id = id;
        Name = name;
        ImagePath = imagePath;
        SongIds = songIds;
    }
}

File: src/types/SongList.ts
export class SongList {
  id?: number
  name: string
  imagePath: string
  songIds: number[]

  constructor(id: number, name: string, imagePath: string, songIds: number[]) {
    this.id = id
    this.name = name
    this.imagePath = imagePath
    this.songIds = songIds
  }
}
